tests/API/Management/ClientsTest.php

```diff
@@ -66,8 +66,8 @@ protected function getAllEntities()
         // Make sure we only have one result, as requested.
         $this->assertEquals(1, count($paged_results));
 
-        // Make sure we only have the 4 fields we requested.
-        $this->assertEquals(count($fields), count($paged_results[0]));
+        // Make sure we only have the 4 fields we requested plus "tenant".
+        $this->assertEquals(count($fields) + 1, count($paged_results[0]));
```

Comment:
why did this change? Can you point out the change that triggered this?

Reply:
I'm pretty sure it switched when the token changed from an SDK-generated one using the global client secret to one obtained with a client credentials grant.

Reply:
Tests shouldn't have changed only because a token value changed, unless the new scope is different from the previous one. Lets check that first and ensure that is the expected behavior.

Reply:
Confirmed. Here is the `GET` call with a "regular" API token (CC grant or manually created in the dashboard):

![Screenshot 2019-07-12 12 19 44](https://user-images.githubusercontent.com/855223/61153195-630c0c00-a49f-11e9-8d95-229039777327.png)

Here is the same exact call with a generated token (what we used before this change):

![Screenshot 2019-07-12 12 18 36](https://user-images.githubusercontent.com/855223/61153154-4079f300-a49f-11e9-9d04-b9a56ef6d4d6.png)